hcl2template/plugin.go

```diff
@@ -64,6 +64,10 @@ func (cfg *PackerConfig) detectPluginBinaries() hcl.Diagnostics {
 		},
 	}
 
+	if runtime.GOOS == "windows" && opts.Ext == "" {
```

Comment:
We might be able to do this once further down the chain but it was not clear to me where else the Ext will be needed. So I opted to duplicate this logic here and in the init command.

Reply:
I think this is good 🙂  !